Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/AssetFolderTableView.cpp

```diff
@@ -31,16 +31,44 @@ namespace AzQtComponents
         if (index != rootIndex())
         {
             QAbstractItemView::setRootIndex(index);
-            emit rootIndexChanged(index);
+            emit tableRootIndexChanged(index);
         }
     }
 
-
     void AssetFolderTableView::SetShowSearchResultsMode(bool searchMode)
     {
         m_showSearchResultsMode = searchMode;
     }
 
+    void AssetFolderTableView::mousePressEvent(QMouseEvent* event)
+    {
+        const auto p = event->pos();
+        if (auto idx = indexAt(p); !idx.isValid())
+        {
+            selectionModel()->clear();
+            return;
```

Comment:
nit: this is a bit small for an early return, can you just make the TableView::mousePressEvent(event) an else clause and remove the return?

Reply:
done!